parm/parm_fv3diag/diag_table_cpl

```diff
@@ -108,8 +108,8 @@
 "gfs_dyn",     "snowwat",     "snmr",         "fv3_history",    "all",  .false.,  "none",  2
 "gfs_dyn",     "rainwat",     "rwmr",         "fv3_history",    "all",  .false.,  "none",  2
 "gfs_dyn",     "graupel",     "grle",         "fv3_history",    "all",  .false.,  "none",  2
-#"gfs_dyn",     "ice_nc",      "nccice",       "fv3_history",    "all",  .false.,  "none",  2
-#"gfs_dyn",     "rain_nc",     "nconrd",       "fv3_history",    "all",  .false.,  "none",  2
+"gfs_dyn",     "ice_nc",      "nccice",       "fv3_history",    "all",  .false.,  "none",  2
+"gfs_dyn",     "rain_nc",     "nconrd",       "fv3_history",    "all",  .false.,  "none",  2
```

Comment:
Not knowing while these were present but commented out before, I don't know whether it is appropriate to uncomment them now or not.

Reply:
These fields were turned on by the request of @yangfanglin as they are needed for Thompson MP.

Reply:
Sounds good.